ocs_ci/helpers/omadops_helpers.py

```diff
@@ -0,0 +1,120 @@
+import uuid
+import time
+import threading
+import random
+import logging
+from datetime import datetime
+
+import tests.OMADOps.conftest as omadops_config
+
+log = logging.getLogger(__name__)
+
+
+def dummy_func(op):
+    log.info(f"Running Op = '{op}' from thread '{threading.currentThread().getName()}'")
+    return True
+
+
+def is_in_timeframe():
+    """
+    Function for determine if test should stop
+
+    Returns:
+        bool: True if END_TIME arrived, False otherwise
+    """
+    return datetime.now() < omadops_config.END_TIME
+
+
+def run_user_ops():
+    """
+    User Ops function:
+      - Runs while in timeframe (configured in tests.OMADOps.config.EXECUTION_TIME_HOURS)
+      - Steps:
+        1. Loop through the list and check if elements are None or thread.is_alive()
```

Comment:
app workloads and user operations have to be running in the background in parallel to the selected random admin ops. Longevity tests automates several user ops and workloads running in the backgrounds. Let's utilize them here as we discussed earlier.